Mobile/Android/Templates/xxx-clearing_account_info_passcode.md

```diff
@@ -0,0 +1,17 @@
+###  Bug: Clearing app data removes passcode 
+
+#### Pr: https://github.com/owncloud/android/pull/XXX 
+
+Devices:
+
+Server: 
+
+
+---
+
+ 
+TestID | Test Case | Steps | Expected Result | Result | Related Comment
+:--------: | :------------- | :------------- | :-------------- | :----- | :------
+| 1 | Clear data with passcode   | 1. Set a passcode in the app<br>2. Go to Settings > Apps > ownCloud > Storage > Manage Space and clear data. <br> 3. Enter the app | The passcode is required. All data (accounts, files, settings...) are removed | 
```

Comment:
With the current behaviour, accounts aren't removed on the device.
The process removes: Settings, Database and Server Certificates, as with the default clean data process.
Downloaded files are on the device too.